addon-test-support/-private/ember-exam-qunit-test-loader.js

```diff
@@ -107,6 +108,17 @@ export default class EmberExamQUnitTestLoader extends TestLoader {
     super.unsee(moduleName);
   }
 
+  /**
+   * setupModuleMetadataHandler() register QUnit callback to enable generating module metadata detail file.
+   */
+  setupModuleMetadataHandler() {
+    this._qunit.moduleDone((details) => {
+      // testem:module-done-detail is sent to server to keep track of test module details.
+      // 'details' contains module metadata like module name, total number of assertion ran in the module, and module runtime.
```

Comment:
should we just name this as `metadata` ?

Reply:
yes I think we should :)